src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="INFO">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss} %-5level %-20logger{1}: %msg%n" charset="UTF-8"/>
+        </Console>
+        <File name="FileLog" fileName="logs/${date:yyyy-MM-dd}-wolneTerminy.log">
+            <PatternLayout pattern="%d{dd-MM-yyyy HH:mm:ss} %-5level %msg%n" charset="UTF-8"/>
+        </File>
+        <File name="FullFileLog" fileName="logs/${date:yyyy-MM-dd}-app.log">
+            <PatternLayout pattern="%d{dd-MM-yyyy HH:mm:ss} %-5level %msg%n" charset="UTF-8"/>
+        </File>
+        <Async name="Async">
+            <AppenderRef ref="FileLog"/>
+            <AppenderRef ref="FullFileLog"/>
+        </Async>
+    </Appenders>
+    <Loggers>
+        <Root level="info">
+            <AppenderRef ref="Console"/>
+            <AppenderRef ref="FullFileLog" level="ALL"/>
+            <AppenderRef ref="FileLog" level="error"/>
```

Comment:
ten nie powinien mieć tylko info?

Reply:
root jest info, więc byłoby to bez sensu, można go nazwać po prostu ErrorFileLog